.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,89 @@
+name: Bug Report
+description: File a bug report.
+title: "[Bug]: "
+labels: ["bug"]
+
+body:
+  - type: markdown
+    attributes:
+      value: |
+        # Thanks for taking the time to fill out this bug report!
+        ### The following information will help us resolve your issue.
+  - type: textarea
+    id: what-happened
+    attributes:
+      label: What happened?
+      description: Also tell us, what did you expect to happen?
+    validations:
+      required: true
+  - type: textarea
+    id: reproduce
+    attributes:
+      label: Steps to Reproduce
+      description: |
+        Please copy and paste the code you were trying to run that caused the error.
+
+        Feel free to include as little or as much as you think is relevant. This section will be automatically formatted into code, so no need for backticks.
+      render: python
+    validations:
+      required: true
+  - type: textarea
+    id: traceback
+    attributes:
+      label: Traceback
+      description: |
+        Please copy and paste the traceback produced by the error.
+
+        This section will be automatically formatted into code, so no need for backticks.
+      render: python
+  - type: dropdown
+    id: os
+    attributes:
+      label: Operating System
+      options:
+        - Windows
+        - macOS
+        - Linux
+    validations:
+      required: true
+  - type: dropdown
+    id: executable
+    attributes:
+      label: Python Executable
+      options:
+        - Conda
+        - Python
+    validations:
+      required: true
```

Comment:
Is this helpful information? I don't think I have ever used this in a debug.

Reply:
For widgets they definitely have less options, we can skip it